tests/gui/test_04_preprocessing.py

```diff
@@ -28,7 +28,7 @@ def test_page_04_loads_with_input():
     at.run()
 
     assert not at.exception
-    assert at.columns[3].selectbox.len == 6
+    assert at.columns[3].selectbox.len == 3
     assert at.button.len == 2
```

Comment:
add a check for the newly added check boxes?

Reply:
This was just to check any interface is loaded. I don't want to tie down the exact widgets too strictly.